docs/Tutorials/Privacy/Private-Transaction-Performance-Best-Practice.md

```diff
@@ -0,0 +1,73 @@
+# Private transaction performance best practices
+
+This document collects deployment and usage hints to help achieving high performance for private transaction.
+If transaction throughput or latency is not meeting your expectations, please consider the following before raising the issue.
+
+## General performance
+
+Private transaction use the same facilities as public ones.
+General Besu performance tunings applies.
+Specific approaches are out of scope of this document.
+However the following item is of such outstanding impact to performance that it is  mentioned here:
```

Comment:
However the following items are of such outstanding impact to performance that they are mentioned here:

Reply:
I had that originally but reverted to singular because the list only contains one element. Should we use plural nonetheless?